### ISSUE TITLE
Integration testing: running three suites simultaneously

### ISSUE THREAD

**Issue by mercmobily** [Polymer/web-component-tester#669]
I have a complex SPA that allows 1) Inter-client chatting 2) Admin interface . Everything is nicely unit-tested and functional-tested. What I am missing is integration testing.

To *actually* run an integration test, I would have to run three different sets of tests _at the same time_:

* User 1 (a client)
* User 2 (a service provider)
* Admin user

The idea is to get these to run via WTC at the same time. So, the database is initially reset and testing starts. User 1 registers as a user, and so does User 2. User 2 applies to become a service provider. Admin user approves it. User 1 indicates that she's looking for a provider. User 2 gets the notification. User 1 and User 2 chat together in real time, and so on and so forth

Basically, I would like to use WTC to run three different instances of the application, and each running different tests, effectively "talking" to each other. We have a very good idea of what the tests will be like, because at the moment three actual *people* do this by following a script, and check that what's happening is totally correct.

What worries me is that while running one test after another is  business as usual when dealing with just one client, it doesn't really work when talking about three different browsers using an app and influencing each other.

Am I being totally crazy? Shall I let those three guys keep doing their bi-weekly tests *by hand*? (I feel like I am kind of torturing them!)
